src/components/playlist-card/PlaylistCard.jsx

```diff
@@ -0,0 +1,33 @@
+import "./PlaylistCard.css";
+import { AiFillDelete } from "../../icons";
+import { removePlaylist } from "../../api/apicalls";
+import { useVideoContext } from "../../context/VideoContext";
```

Comment:
You can implement `barrel` import / export

Reply:
will try to implement it  next time.